src/styles/blocks/header.scss

```diff
@@ -0,0 +1,62 @@
+
+.head {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__logo {
+    align-self: center;
+    margin-left: 50px;
+    height: 40px;
+  }
+
+}
+
+.nav {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-right: 50px;
+
+  &__list {
+    display: flex;
+    list-style-type: none;
+    padding-left: 0;
+    margin: 0;
+  }
+
+  &__link {
+    position: relative;
+    display: inline-block;
+    color: $black_text_color;
+    text-transform: uppercase;
+    margin: 0 10px;
+    padding: 23px 0;
+  }
+
+  .is-active {
+    color: $blue_pop_color;
+    margin-left: 0;
+  }
+
+  &_last {
+    margin-right: 0;
+  }
+
+  .is-active::after {
+    position: absolute;
+    bottom: 0;
+    display: block;
+    content: "";
+    width: 36px;
+    height: 4px;
+    background-color: $blue_pop_color;
+    border-radius: 8px;
+  }
```

Comment:
Check the 9th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)